src/libostree/ostree-kernel-args.c

```diff
@@ -166,6 +193,9 @@ _ostree_kernel_args_append_proc_cmdline (OstreeKernelArgs *kargs,
   g_autofree char *proc_cmdline = NULL;
   gsize proc_cmdline_len = 0;
   g_auto(GStrv) proc_cmdline_args = NULL;
+  char *filtered_prefixes[] = { "BOOT_IMAGE=",
```

Comment:
Can you add a comment here for each one, just like

```
{ "BOOT_IMAGE=", /* GRUB2 */
  "initrd=", /* sd-boot */
```

And also a comment at the top like:

```
/* If you update the list, also change the 
   lists in the tests; e.g.  tests/test-admin-deploy-karg.sh 
*/
```